oak_containers/proto/interfaces.proto

```diff
@@ -33,6 +33,7 @@ message GetApplicationConfigResponse {
   bytes config = 1;
 }
 
+// Defines the service exposed by the launcher, that can be invoked by the orchestrator.
```

Comment:
nit: I think the service is also called by stage 1

Reply:
+1

Reply:
indeed, good catch